Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+#!groovy​
+
+properties([pipelineTriggers([cron('H * * * *')])])
+
+pipeline {
+    agent none
+
+    stages {
+        stage('clean') {
+            agent { label 'master' }
+            steps {
+                sh 'git clean -fdx'
+            }
+        }
+
+        stage('sync') {
+            agent {
+                dockerfile {
+                    additionalBuildArgs '--build-arg BUILDER_UID=${JENKINS_UID:-9999}'
+                }
+            }
+            steps {
+                sh 'aws s3 sync --delete . s3://${BUCKET} --exclude ".git/*" --exclude "*.md" --exclude ".gitignore" --exclude "Jenkinsfile" --exclude "Dockerfile"'
```

Comment:
if this PR is to be merged then the two new files will need to be excluded on the preexisting jenkins sync_s3 job as well

Reply:
Yes good point